warehouse/templates/pages/help.html

```diff
@@ -184,7 +184,17 @@ <h2>Problems</h2>
 
         <h3 id="login-problem">{{ login_problem() }}</h3>
         <p>
-          You should be able to <a href="{{ request.route_path('accounts.reset-password') }}">reset your password</a> on the site; you'll receive an email with a password reset link. If you no longer have access to the email address associated with your account, <a href="https://github.com/pypa/warehouse/issues" target="_blank" rel="noopener">report it to us on GitHub</a>, so we can try to help.
+          If you've forgotten your PyPI password but you remember your email address, follow these steps to get a new password:
+         <ol>
+  	    <li>Go to <a href="{{ request.route_path('accounts.reset-password') }}">reset your password</a>.</li>
+  	    <li>Enter the email address you use for PyPI and click Request password reset.</li>
```

Comment:
"email address _or username_", and let's change "click Request password reset" to "submit the form".